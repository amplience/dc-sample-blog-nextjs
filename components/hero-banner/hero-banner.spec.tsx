/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import HeroBanner from './hero-banner';

describe('HeroBanner', () => {
  test('renders hero banner with title and subtitle', async () => {
    const props = {
      title: 'Test title',
      subTitle: 'Test subtitle'
    };
    const component = await renderer.create(<HeroBanner {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });

  test('renders hero banner with title only', async () => {
    const props = {
      title: 'Test title'
    };
    const component = await renderer.create(<HeroBanner {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
