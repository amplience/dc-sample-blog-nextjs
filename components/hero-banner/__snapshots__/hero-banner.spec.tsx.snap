// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroBanner renders hero banner with title and subtitle 1`] = `
<section
  className="jsx-607688792"
>
  <h1
    className="jsx-641291359"
  />
</section>
`;

exports[`HeroBanner renders hero banner with title only 1`] = `
<section
  className="jsx-607688792"
>
  <h1
    className="jsx-641291359"
  />
</section>
`;
